app/views/posts/new.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1 class= "create-post">Create a post</h1>
+<%= form_with_url: user_posts_path, scope: :data do |form| %>
+<section class="post-form">
+  <div class="title-box">
```

Comment:
- [ ] Let's fix this error, I couldn't go ahead with the review because of the error in the image below.

![form](https://user-images.githubusercontent.com/53124548/156522710-c3ac5905-a432-4075-8cad-06b6ee5dda8c.png)